dockerfiles/csci4210/ubuntu22.04/Dockerfile

```diff
@@ -19,3 +19,16 @@ RUN apt-get update \
     && apt-get -y --no-install-recommends install gcc g++ \
     && rm -rf /var/lib/apt/lists/*
 
+# Install Valgrind
+RUN apt-get update \
+    && apt-get -y --no-install-recommends install valgrind \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install pip
+RUN apt-get update \
+    && apt-get -y --no-install-recommends install python3-pip \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install numpy
+RUN pip3 install numpy 
```

Comment:
pip isn't required at runtime of the container, thus it should probably be removed. The best way to do this would be to make a single RUN command which installs pip, install numpy, and uninstall pip. This would be most optimal for the layer size.

Reply:
Good point. It is fun to be back doing instead of grading/answering. How does this one look? I did the single command pretty much as you suggested, and tested that pip3 is actually removed.